### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Whytehorse**
Added bill of rights to constitution as article 13

### CODE REVIEW

**Comment by xsttx** [comment]
@Whytehorse all of these are very different from each other. I suggest you do pull request for one at the time, under relevant articles, otherwise it's nearly impossible to comment on or merge.

**Comment by Whytehorse** [comment]
@xstt What? It's one article and it's new

**Comment by Whytehorse** [comment]
it's the universal declaration of human rights put into simple english. It's one article called rights. Please pull

**Comment by Whytehorse** [comment]
I added insurance dApp folder and readme plus sample code

**Comment by ETeissonniere** [comment]
@Whytehorse that content doesn't fit the "update readme" title, can you please different pull req for the different part? Thanks.

File: dapp/src/insurance/testflightdelay.sol [hunk 1]
```diff
@@ -0,0 +1,745 @@
+/*
+
+	FlightDelay with Oraclized Underwriting and Payout
+	All times are UTC.
+	Copyright (C) Christoph Mussenbrock, Stephan Karpischek
+	
+*/
```
Comment (ETeissonniere) [inline dapp/src/insurance/testflightdelay.sol:7]: Seems to have been copy pasted from an other repo, better to put a link to that document in dapp/src/insurance/readme.md

File: dapp/src/insurance/testflightdelay.sol [hunk 2]
```diff
@@ -0,0 +1,745 @@
...
+
+
+import "./github.com/oraclize/ethereum-api/oraclizeAPI.sol";
+import "./github.com/Arachnid/solidity-stringutils/strings.sol";
```
Comment (ETeissonniere) [inline dapp/src/insurance/testflightdelay.sol:11]: The following imports are not valid, they are loading files from a directory which is not included

Comment (ETeissonniere) [reply dapp/src/insurance/testflightdelay.sol:11]: It is also highly recommended not to import code from URLs, those files could be modified by their owner without the end user noticing it

File: dapp/src/insurance/testflightdelay.sol [hunk 3]
```diff
@@ -0,0 +1,745 @@
...
+	uint8 constant maintenance_Emergency 	= 255;
+	
+	
+	// gas Constants for oraclize
+	uint constant oraclizeGas 				= 500000;
+
+	// URLs and query strings for oraclize
+
+	string constant oraclize_RatingsBaseUrl =
+		"[URL] json(https://api.flightstats.com/flex/ratings/rest/v1/json/flight/";
```
Comment (ETeissonniere) [inline dapp/src/insurance/testflightdelay.sol:186]: The whole contract makes use of oraclize to perform external calls, I am sure there is a way to do everything on chain and in a simpler way

**Review by Whytehorse — COMMENTED** [review]
test review

**Pull request closed without merging**